compiler/aarch64/codegen/OMRTreeEvaluator.cpp

```diff
@@ -2499,7 +2499,80 @@ OMR::ARM64::TreeEvaluator::vmdivEvaluator(TR::Node *node, TR::CodeGenerator *cg)
 TR::Register*
 OMR::ARM64::TreeEvaluator::vmfmaEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   return TR::TreeEvaluator::unImpOpEvaluator(node, cg);
+   TR::Node *firstChild = node->getFirstChild();
+   TR::Node *secondChild = node->getSecondChild();
+   TR::Node *thirdChild = node->getThirdChild();
+   TR::Node *fourthChild = node->getChild(3);
+
+   TR::Register *firstReg  = cg->evaluate(firstChild);
+   TR::Register *secondReg  = cg->evaluate(secondChild);
+   TR::Register *thirdReg  = cg->evaluate(thirdChild);
```

Comment:
I don't think you need two whitespaces between `Reg` and `=` in these three lines.

Reply:
Agreed. Fixed it.